File: snippets/fsharp/System/NullReferenceException/Overview/Array2.fs
module Array2

// <Snippet9>
open System

let values = [| "one"; null; "two" |]
for i = 0 to values.GetUpperBound 0 do
    printf $"""{if values[i] <> null then values[i].Trim() else ""}{if i = values.GetUpperBound 0 then "" else ", "}"""
Console.WriteLine()
// The example displays the following output:
//       one, , two
// </Snippet9>

File: snippets/csharp/System/NullReferenceException/Overview/Array2.cs
using System;

public class Array2Example
{
    public static void Main()
    {
        // <Snippet9>
        string[] values = [ "one", null, "two" ];
        for (int ctr = 0; ctr <= values.GetUpperBound(0); ctr++)
            Console.Write("{0}{1}",
                          values[ctr] != null ? values[ctr].Trim() : "",
                          ctr == values.GetUpperBound(0) ? "" : ", ");
        Console.WriteLine();

        // The example displays the following output:
        //       one, , two
        // </Snippet9>
    }
}
